SPECS/rpm/rpm.spec

```diff
@@ -138,7 +128,6 @@ sed -i 's/extra_link_args/library_dirs/g' python/setup.py.in
 make %{?_smp_mflags}
 
 pushd python
-python2 setup.py build
 python3 setup.py build
```

Comment:
Nit: while you're here (and in other specs), we can change this to `%py3_build` for better consistency.